targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_LPC/analogin_api.c

```diff
@@ -49,7 +49,7 @@ void analogin_init(analogin_t *obj, PinName pin)
     ADC_ClockPower_Configuration();
 
     /* Ensure the ADC clock derived from the system clock is less than 80MHz */
```

Comment:
The comments are not matching the code.

Reply:
The system clock is managed as BusClock by the internal clocking code structure :). Comment is correct but code needs to be changed.